#request fields
bool enable
float32 speed
---
#response fields
string res
